content/operate/rs/security/_index.md

```diff
@@ -11,59 +11,22 @@ weight: 60
 aliases:
     - /operate/rs/administering/designing-production/security
 ---
-Security is an important part of any production system. This section describes the security features and settings available in Redis Enterprise.
 
-## Architecture security
+Redis Enterprise Software provides various features to secure your Redis Enterprise Software deployment:
 
-When deploying Redis Enterprise Software to production, we recommend the following practices:
+| Authentication | Authorization | Encryption | TLS |
```

Comment:
@rrelledge please modify according to the suggested table in the doc (https://docs.google.com/document/d/1wYysLIidXybTvfX_RYg7YgWjDpayMa3RmBHRGFAoQNs/edit?usp=sharing)